mobilenet/model_info.py

```
@@ -0,0 +1,37 @@
+try:
+    img = Image.open(img_path)
+    weights = MobileNet_V2_Weights.IMAGENET1K_V1
+    preprocess = weights.transforms()
+    input_tensor = preprocess(img).unsqueeze(0).to(DEVICE)
+
+    with torch.no_grad():
+        features = model(input_tensor)
+
+    print(f"Input image: {img_path}")
+    print(f"Feature vector shape: {features.shape}")
+    print(f"Feature vector (first 20 elements):")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)